src/wazuh_modules/vulnerability_scanner/src/scanOrchestrator/cveSolvedInventorySync.hpp

```diff
@@ -48,60 +48,64 @@ class TCVESolvedInventorySync final
      */
     std::shared_ptr<TScanContext> handleRequest(std::shared_ptr<TScanContext> data) override
     {
-        // We look for the CVEs in the agent's vulnerabilities inventory
-        const auto& column = AFFECTED_COMPONENT_COLUMNS.at(AffectedComponentType::Package);
-
-        for (const auto& [key, value] : TInventorySync<TScanContext>::m_inventoryDatabase.seek(data->agentId(), column))
+        // We look for the CVEs in the agent's vulnerabilities inventory for OS and Packages
```

Comment:
We should discuss with the team about this use case.
I understand that it wasn't in the requirements of
https://github.com/wazuh/wazuh/issues/23066

A hotfix insertion will eventually be reflected for the OS CVEs in the next OS scan. But it isn't a bad idea to react faster.

Reply:
The requirements were too open, I had to test as much as possible the behavior of the scanner with fixed content. 

About this 

> A hotfix insertion will eventually be reflected for the OS CVEs in the next OS scan. But it isn't a bad idea to react faster.

But that new OS scan won't happen after a restart? we are ignoring OS delta events for obvious reason. 

I'll let this as it is now, until we define whether or not I should remove this. 

CC: @Dwordcito

Reply:
> The requirements were too open, I had to test as much as possible the behavior of the scanner with fixed content.

You've selected a wide range of tests.
I meant that when we developed that feature, we focused on the packages affected by the hotfixes only.

> But that new OS scan won't happen after a restart? we are ignoring OS delta events for obvious reason.

The OS should be re-scanned in every syscollector scan.

> I'll let this as it is now, until we define whether or not I should remove this.

Sure, we wait for definitions.

Reply:
>  I meant that when we developed that feature, we focused on the packages affected by the hotfixes only.

Did we really focus on that? or did we forget to update it there after we changed it here https://github.com/wazuh/wazuh/pull/23088?